pandas/tests/test_groupby.py

```diff
@@ -442,6 +442,12 @@ def test_grouper_creation_bug(self):
         result = g.sum()
         assert_frame_equal(result, expected)
 
+        # GH14334
+        # pd.Grouper(key=...) may be passed in a list
+        g = df.groupby([pd.Grouper(key='A')])
+        result = g.sum()
+        assert_frame_equal(result, expected)
+
```

Comment:
can you add a test with multiple ``pd.Grouper`` passed (both with keys) and then a string and a ``Grouper`` (unless these combinations already exist)